### ISSUE TITLE
Update to 2020 GCB release?

### ISSUE THREAD

**Issue by danielhuppmann** [openclimatedata/global-carbon-budget#7]
This is a really nice resource - do you plan to update it for the GCB 2020 release?

Also, I would be interested to hook this up to the https://pyam-iamc.readthedocs.io (which would be easiest if you turn this into an installable package or even release it via pypi).

**Comment by rgieseke** [comment]
Yes, already downloaded and started cleaning scripts and dependencies ...
